src/rpcblockchain.cpp

```diff
@@ -706,9 +709,10 @@ UniValue getblockchaininfo(const UniValue& params, bool fHelp)
     bip9_softforks.push_back(BIP9SoftForkDesc("segwit", consensusParams, Consensus::DEPLOYMENT_SEGWIT));  // MVF-BU added
     obj.push_back(Pair("softforks",             softforks));
     obj.push_back(Pair("bip9_softforks", bip9_softforks));
+    obj.push_back(Pair("difficultyadjinterval", consensusParams.DifficultyAdjustmentInterval(tip->nHeight))); // MVF-BU
 
     // MVF-BU begin output hardfork description (MVHF-BU-DES-TRIG-9)
-    if (!isMVFHardForkActive)
+    if (!isMVFHardForkActive && consensusParams.MVFisWithinRetargetPeriod(tip->nHeight))
```

Comment:
This test determines whether the 'hardforks' section is output.
Initially, I had it set so that it would only output when the hardfork was not active (so one would get the countdown etc.).
But now I'm thinking it should output before the hardfork, and until the hardfork is completely "over", i.e. the end of the retargeting recovery period.

I think that would correspond to

!isMVFHardForkActive || consensusParams.MVFisWithinRetargetPeriod(tip->nHeight)